content/shared/influxdb3-cli/update/table/_index.md

```diff
@@ -0,0 +1,74 @@
+The `influxdb3 update table` command updates an existing table in a database in your {{< product-name >}} instance.
```

Comment:
Table retention periods are only supported in Enterprise. Is this command included in the Core CLI?